src/test/scala/com/microsoft/hyperspace/actions/CreateActionTest.scala

```diff
@@ -104,4 +108,57 @@ class CreateActionTest extends SparkFunSuite with SparkInvolvedSuite {
     assert(
       ex.getMessage.contains(s"Another Index with name ${indexConfig.indexName} already exists"))
   }
+
+  test("Verify rootPaths for given LogicalRelations") {
+    val path1 = sampleParquetDataLocation + "table1"
+    val path2 = sampleParquetDataLocation + "table2"
+    FileUtils.delete(new Path(path1))
+    FileUtils.delete(new Path(path2))
```

Comment:
Can we do
```scala
    withTempPath { p =>
      val path1 = p + "table1"
      val path2 = p + "table2"
```
? You need update this class with `with SQLHelper`.

Reply:
With this fix, `rootPaths` has `file:/` prefix:

```
List("file:/private/var/folders/sd/ynbvcl0120g_16_wm0g56zc40000gn/T/spark-2fef90f7-9b0e-4a67-a3f4-bb158d704fcdtable1")
did not equal
List("/private/var/folders/sd/ynbvcl0120g_16_wm0g56zc40000gn/T/spark-2fef90f7-9b0e-4a67-a3f4-bb158d704fcdtable1") (CreateActionTest.scala:163)
[info]
```